ludicrousdb/includes/class-ludicrousdb.php

```diff
@@ -1789,33 +1794,31 @@ public function check_tcp_responsiveness( $host, $port, $float_timeout ) {
 			return false;
 		}
 
-		// Defaults
+		// Defaults.
 		$errno  = 0;
 		$errstr = '';
 
-		// Try to get a new socket
-		$socket = ( WP_DEBUG )
-			? fsockopen( $host, $port, $errno, $errstr, $float_timeout )
-			: @fsockopen( $host, $port, $errno, $errstr, $float_timeout );
+		// Try to get a new socket.
+		$socket = ( WP_DEBUG ) ? fsockopen( $host, $port, $errno, $errstr, $float_timeout ) : @fsockopen( $host, $port, $errno, $errstr, $float_timeout ); // phpcs:ignore WordPress.PHP.NoSilencedErrors.Discouraged
```

Comment:
Multiline ternaries need to be allowed. This is not easier to read now than it was before.